### PR TITLE
Add Team Timnit to the sprint teams page

### PR DESCRIPTION
**Pull request by eileen-nava**
Change Notes:
- Added Team Timnit to the sprint teams page

Screenshots
Screen Shot 2023-05-25 at 10 40 51 AM

### CODE REVIEW

**Comment by eileen-nava** [comment]
Note: I won't merge this until confirming that it's okay to link to the readme, as that is currently a work in progress.

**Review by zachmargolis — APPROVED** [review]
LGTM

**Comment by eileen-nava** [comment]
@zachmargolis Could you please look at this again?  I linked to the charter instead of the readme and now the spacing is off.  I can't figure out what caused it.  TY!

File: _articles/sprint-teams.md [hunk 1]
```diff
@@ -32,5 +32,8 @@ This sprint team is not currently active
 {% if sprint_team.readme -%}
 * **Readme**: [View Team {{sprint_team.name}} Readme]({{ sprint_team.readme }})
 {%- endif %}
+{% if sprint_team.charter -%}
+* **Charter**: [View Team {{sprint_team.name}} Charter]({{ sprint_team.charter }})
+{%- endif %}
```
Comment (NavaTim) [inline _articles/sprint-teams.md:37]: I expect the issue in the rendering may be due to two consecutive newlines left over by the `{%-endif %}..{% if -%}` clauses here. We may be able to handle this by either containing the newline in the conditional or switching to `elsif`.

Also see:
- [Liquid > Tags > Control Flow > elsif / else](https://shopify.github.io/liquid/tags/control-flow/#elsif--else)
- [Liquid > Basics > Whitespace control](https://shopify.github.io/liquid/basics/whitespace/)

One of these may fix the issue:

```suggestion
{%- if sprint_team.readme %}
* **Readme**: [View Team {{sprint_team.name}} Readme]({{ sprint_team.readme }})
{%- endif -%}{%- if sprint_team.charter %}
* **Charter**: [View Team {{sprint_team.name}} Charter]({{ sprint_team.charter }})
{%- endif %}
```

```suggestion
{% if sprint_team.readme -%}
* **Readme**: [View Team {{sprint_team.name}} Readme]({{ sprint_team.readme }})
{% elsif sprint_team.charter -%}
* **Charter**: [View Team {{sprint_team.name}} Charter]({{ sprint_team.charter }})
{%- endif %}
```

**Comment by eileen-nava** [comment]
@zachmargolis I hear what you're saying.  I wanted to follow product's request to link to the team charter, and linking to the team charter from the readme section seems misleading.

File: _articles/sprint-teams.md (hunk 1 above)
Comment (eileen-nava) [reply _articles/sprint-teams.md:37]: That worked, ty! 🙏🏻

**Comment by eileen-nava** [comment]
@zachmargolis New update: I asked product about it in standup and they are fine with linking to the charter from the readme section.  I reverted the changes to the template. 👍🏻

**Review by zachmargolis — APPROVED** [review]

**Pull request merged**